### PR TITLE
Create base/stage models (iteration)

### PR DESCRIPTION
**Pull request by zaharenka**
Pull request contains the same performed steps for new entities as described in #17

### CODE REVIEW

File: models/staging/edfi_3/base/base_ef3__peoples.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+with peoples as (
```
Comment (maxalmazov) [inline models/staging/edfi_3/base/base_ef3__peoples.sql]: `people` is already plural

Comment (holdorph) [reply models/staging/edfi_3/base/base_ef3__peoples.sql]: It should match what the API is.  and the API is /ed-fi/people

Comment (zaharenka) [reply models/staging/edfi_3/base/base_ef3__peoples.sql]: @maxalmazov @holdorph that makes sense. I got caught 😅

**Comment by holdorph** [comment]
Try to add schoolYearTypes to this PR.  You can find it currently in PR #12 and #14.

File: models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql
Comment (holdorph) [inline models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql:1]: Please roll these changes back.  If we have no content changes, lets not change the whitespace for a file we did not originally create.

Comment (holdorph) [reply models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql:1]: I might have been mistaken here.  I am trying to find where (and why) this file was introduced.

Comment (holdorph) [reply models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql:1]: Do we have any jsonl data for this entity?

**Review by holdorph — APPROVED** [review]

File: models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql
Comment (holdorph) [reply models/staging/edfi_3/base/base_ef3__post_secondary_institutions.sql:1]: This was added by a previous PR by our team, though we have no jsonl.  Decision (in chat) was to leave this file in.

File: models/staging/edfi_3/stage/stg_ef3__credentials.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (vzakharko) [inline models/staging/edfi_3/stage/stg_ef3__credentials.sql]: @holdorph @zaharenka  sometimes we have an empty line at the beginning of the file, sometimes not, maybe it makes sense to do it uniformly?

Comment (holdorph) [reply models/staging/edfi_3/stage/stg_ef3__credentials.sql]: Where it's noticed, I would remove it.  All of the files from the original repository (not created by us) do NOT start with a blank line.

**Review by vzakharko — APPROVED** [review]

**Pull request merged**